src/foam/dao/Relationship.js

```diff
@@ -293,6 +293,8 @@ foam.CLASS({
       */
     },
 
+    // TODO(markdittmer): Use of "forward" here doesn't seem right. No call
+    // sites pass a second parameter.
```

Comment:
This was used by *:1 support, which I've removed (and plan to put back, but haven't yet). I've pushed a CL to remove 'forward'.